README.md

```diff
@@ -80,26 +85,34 @@ cp -i config/example_config.yml ~/.dp-cli-config.yml
 vi ~/.dp-cli-config.yml
 ```
 
-[ set paths for:
+update the paths and ssh-user: 
 
 ```yaml
-    dp-setup-path:
-    dp-hierarchy-builder-path:
-    dp-code-list-scripts-path:
-
-   set your `ssh-user:`
+    dp-setup-path: path to your local dp-setup
+    dp-ci-path: path to your local dp-ci
+    dp-hierarchy-builder-path: path to your local dp-hierarchy-builder-path
+    dp-code-list-scripts-path: path to your local dp-code-list-scripts-path
+    ssh-user: Your AWS account name
 ```
 
-and if this is a first time setup, comment out production from environments, thus:
+and if this is a first time setup, comment out `prod` from environments, thus:
 
 ```yaml
-    # - name: production
-    #   profile: production
+     #- name: prod 
+     #  profile: dp-prod
+     #  user: ubuntu 
+     #  tag: awsb
 ```
 
-]
+*Note*: **ssh-user** is a string used to put your name against SecurityGroup changes.
+
+You should receive credentials as part of onboarding. If you do not have credentials yet, please 
+get in touch with the platform team to get an account created
+
+- The account must be registered to an ONS email address (ons.gov.uk or ext.ons.gov.uk)
+- AWS accounts are now managed by the Cloud Ops team (in DST) and are requested in ServiceNow
+- You should receive an email with a link to register
```

Comment:
I think these details should be in the guide about setting up aws access and linked to from here

Reply:
Shall I create the guide in dp-operations or to dp/guides?